### PR TITLE
Update docker doc

### PR DESCRIPTION
**Pull request by nd-02110114**
I update docker environments. (I stop having separate images)
 
 - deepchemio/deepchem:x.x.x
   - Image built by `docker/conda-forge/Dockerfile`
   - This image is built when we push x.x.x. tag
 - deepchemio/deepchem:latest
   - Image built by `docker/master/Dockerfile`
   - This image is built every time we commit to the master branch

I can not check the docker env with GPU. After I checking it, this PR is ready to merge (I will check it in this weekend)

### CODE REVIEW

File: scripts/install_deepchem_conda.sh [hunk 1]
```diff
@@ -36,4 +36,4 @@ conda install -y -q -c deepchem -c rdkit -c conda-forge -c omnia \
     pytest \
     pytest-cov \
     flaky
-yes | pip install -U tensorflow tensorflow-probability
+yes | pip install -U tensorflow==2.2 tensorflow-probability==2.2
```
Comment (nd-02110114) [inline scripts/install_deepchem_conda.sh:39]: I fixed the TF version.

**Comment by rbharath** [comment]
Thanks for the updates! Having regular docker builds will make things much nicer for our users :)

The test failure looks to be unrelated. Going to go ahead and merge in

**Comment by rbharath** [comment]
Oh my bad, just noticed your comment that you want to check the docker GPU build before merging this in. Will leave this open until you're done!

**Comment by nd-02110114** [comment]
I checked the docker GPU build with GCP. This PR is ready to merge!

And, is it better to post the blog about how to use the GCP with DeepChem like [Deploying a DeepChem Sagemaker Notebook Instance](https://forum.deepchem.io/t/deploying-a-deepchem-sagemaker-notebook-instance/174)

**Comment by rbharath** [comment]
@nd-02110114 This is great! Going to go ahead and merge :)

Also, a blog post about how to use DeepChem with GCP would be awesome!

**Pull request merged**